queue/group_remote.go

```diff
@@ -108,7 +116,7 @@ func NewRemoteQueueGroup(ctx context.Context, opts RemoteQueueGroupOptions) (amb
 				case <-ctx.Done():
 					return
 				case <-ticker.C:
-					_ = g.Prune(ctx)
+					grip.Error(errors.Wrap(g.Prune(pruneCtx), "problem pruning remote queue group database"))
```

Comment:
why not collect this and propogate out? catchers are thread safe. 

also message.WrapError() is probably also good/better?

Reply:
This goroutine keeps running after the constructor exits.